### ISSUE TITLE
history tracking

### ISSUE THREAD

**Issue by mahdi00021** [ojkalam/gps_attendance#1]
hi
does have your project tracking on map with select device?

like as this picture:

![download 1](https://user-images.githubusercontent.com/9013165/44628526-716c3a80-a956-11e8-9a85-0ad2310a9613.jpg)

[like as this video traccar web](https://www.youtube.com/watch?v=B8rrqRfvOj4)

**Comment by ojkalam** [comment]
No, it will just track your current location(address) and save it to DB
